### PR TITLE
ComputedValues.h: minor optimizations

### PR DESCRIPTION
**Pull request by shlyakpavel**
See individual commits

### CODE REVIEW

File: Libraries/LibWeb/CSS/ComputedValues.h [hunk 1]
```diff
@@ -719,7 +719,7 @@ class MutableComputedValues final : public ComputedValues {
         m_inherited = static_cast<MutableComputedValues const&>(other).m_inherited;
     }
 
-    void set_aspect_ratio(AspectRatio aspect_ratio) { m_noninherited.aspect_ratio = aspect_ratio; }
+    void set_aspect_ratio(AspectRatio const& aspect_ratio) { m_noninherited.aspect_ratio = aspect_ratio; }
```
Comment (yyny) [inline Libraries/LibWeb/CSS/ComputedValues.h:722]: The old code would do copy-ellision, while the new code will make a copy, which seems to be the opposite of what's intended. This is true for every line in this commit.

Comment (trflynn89) [inline Libraries/LibWeb/CSS/ComputedValues.h:722]: We should go the other way - let callers `move` the argument into this function call, and `move` the argument into the class member:
```c++
void set_aspect_ratio(AspectRatio aspect_ratio) { m_noninherited.aspect_ratio = move(aspect_ratio); }
```

**Review by yyny — CHANGES_REQUESTED** [review]

File: Libraries/LibWeb/CSS/ComputedValues.h (hunk 1 above)
Comment (shlyakpavel) [reply Libraries/LibWeb/CSS/ComputedValues.h:722]: Got you

Comment (shlyakpavel) [reply Libraries/LibWeb/CSS/ComputedValues.h]: I have edited it as suggested by @trflynn89

**Pull request merged**